test/system/practical_support_behaviors_test.rb

```diff
@@ -105,6 +105,29 @@ class PracticalSupportBehaviorsTest < ApplicationSystemTestCase
       end
     end
   end
+
+  describe 'hiding practical support' do
+    before do
+      @patient.practical_supports.create support_type: 'lodging',
+                                         source: 'Other funds (see notes)',
+                                         created_by: @user
+      go_to_practical_support_tab
+    end
+
+    it 'toggles the showing of the practical support tab' do
+      assert has_text? /Practical Support/i
+      c = Config.find_or_create_by(config_key: 'hide_practical_support')
+      c.config_value = { options: ["Yes"] }
+      c.save!
+      page.refresh
+      assert has_no_text? /Practical Support/i
+      c.config_value = { options: ["No"] }
```

Comment:
I don't think we need to test this toggle back and forth, should be covered by the model tests. Can you delete this line thru 127?